### ISSUE TITLE
UseProcessed

### ISSUE THREAD

**Issue by mtrnka** [lujonathanh/PeakSeeker#5]
Can you explain the UseProcessed option?  I think if set to 0, then the processing only affects the peak detection, and when set to 1 it affects the charge state determination.  However, I don’t have a feeling for which is better and why.  If I turn it on, some of my deconvoluted masses shift by 1000s of Da, and it is not clear to me which is a more correct measurement.

**Comment by lujonathanh** [comment]
You are correct: the UseProcessed option affects the charge state determination in that the processed spectrum is used to estimate peak parameters like intensity and m/z, while otherwise the charge determination would use the parameters off the original spectrum. Useprocessed changes the original peak parameters, so it's less accurate. I would only use it with spectra with very high amounts of noise. I have added this comment to the Options file.
